src/components/forms/JurisdictionMetadata/tests/index.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { render } from '@testing-library/react';
```

Comment:
I think we should add tests that explore how the form works:

- when showing errors
    - what errors are actually supposed to be shown to the user
- when submitting
    - is the payload built successfully based on what was uploaded
    - is the API endpoint called with the right params